### DISCUSSION TITLE
how to have access to a nested object from the api response object?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2380] [Help]
Hello,

I am using the api for the data and I am facing a little problem...
{
  "count": 1,
  "next": null,
  "previous": null,
  "results": [
    {
      "id": 4,
      "order": "4.00000000000000000000",
      "code": "ed65bgh36",
      "paye": false,
      "prix": "23.00",
      "nom": "red",
      "contrats": [],
      "date_payement": null,
      "date_creation": "2021-08-17T09:41:45.670585Z",
      "document": "https://budi.senocoschool.com//prod-budi-app-assets/app_573d13a984da40799302a2094066860d/attachments/f4e9f8a0-53ee-4ad9-bb8e-4ea343d7f095.json"
    }
  ]
}

How can I have access to the results only to define the schema ?

**Comment by mjashanks** [comment]
Hey Oswald. Unfortunately, it's not possible at the moment. We have a feature planned for this: https://github.com/Budibase/budibase/issues/2093 - can't promise a date for this right now.

**Reply by oswaldB** [reply]
I changed my backend to get the right infos and I cannot succeed in showing the element on the designer.
![image](https://user-images.githubusercontent.com/3023526/129707859-9d07f95c-1a7c-439e-b5ac-4fcc8b689e77.png)

**Reply by mjashanks** [reply]
Hey Oswald, I tested this and it worked ok for me. I used the GitHub API:

![image](https://user-images.githubusercontent.com/3524181/129863577-0636522c-4bfc-4240-8b14-52006b55e2e1.png)

Then, I tested it using a Data Provider + Repeater + Headline

![image](https://user-images.githubusercontent.com/3524181/129863728-759be26d-4ff2-43cb-8c98-06e0398e91b0.png)

Are you positive that your API is returning an array of items?
